### PR TITLE
Bug fix/long message label issue72

### PR DESCRIPTION
**Pull request by Summa-Labs**

### CODE REVIEW

File: Sources/MessagesViewController.swift [hunk 1]
```diff
@@ -284,6 +284,7 @@ extension MessagesViewController {
         case Notification.Name.UIKeyboardDidShow:
             // Only runs once
             messagesCollectionView.contentInset =  UIEdgeInsets(top: topLayoutGuide.length, left: 0, bottom: messageInputBar.frame.height, right: 0)
+			messagesCollectionView.contentOffset = CGPoint(x: 0, y: messagesCollectionView.contentSize.height-messagesCollectionView.frame.height+50)
```
Comment (Sherlouk) [inline Sources/MessagesViewController.swift:287]: Indented a bit too far, and where has this 50 come from?

File: Sources/MessageLabel.swift [hunk 2]
```diff
@@ -155,6 +155,7 @@ open class MessageLabel: UILabel, UIGestureRecognizerDelegate {
         // Message Label Specific
         self.numberOfLines = 0
         self.lineBreakMode = .byWordWrapping
+		self.font = UIFont.defaultMessageLabelFont()
```
Comment (Sherlouk) [inline Sources/MessageLabel.swift:158]: Indented a bit too far!

File: Sources/UIFont+Extensions.swift [hunk 3]
```diff
@@ -0,0 +1,16 @@
+//
```
Comment (Sherlouk) [inline Sources/UIFont+Extensions.swift:1]: Copyright header needs replacing with the same one from other files (includes our full license)

**Review by Sherlouk — CHANGES_REQUESTED** [review]
Couple minors, haven't had a chance to run it fully yet to ensure it actually works!

Also woo for first time contributor! Welcome!

**Comment by SD10** [comment]
@Summa-Labs The real issue is I forgot to apply the font in: `apply(_ layoutAttributes: UICollectionViewLayoutAttributes)` 😅 

If you want to make this patch on the `enhance/media-messages` branch for `TextMessageCell` you're welcome to.

**Comment by SD10** [comment]
@Summa-Labs Thank you for finding this bug I will take care of it 👍

**Pull request closed without merging**